integration-tests/api/__tests__/admin/order-edit.js

```diff
@@ -2024,7 +2024,7 @@ describe("[MEDUSA_FF_ORDER_EDITING] /admin/order-edits", () => {
       )
 
       response = await api.get(
```

Comment:
comment: If you could remove the get entirely as the update line item will return the order edit with the default relations which now includes the adjustments